### ISSUE TITLE
java.lang.NoClassDefFoundError: Could not initialize class jenkins.model.Jenkins

### ISSUE THREAD

**Issue by richardlau** [nodejs/build#2510]
@Trott has reported in Slack that several hosts in the CI are failing with a Java exception, "java.lang.NoClassDefFoundError: Could not initialize class jenkins.model.Jenkins", and has rebooted several of the machines :heart:.

Remaining platforms are Windows and the docker Linux containered hosts.

**Comment by richardlau** [comment]
I've marked the following Windows hosts offline (cc @joaocgreis )
- https://ci.nodejs.org/computer/test-rackspace-win2012r2_vs2019-x64-1/
- https://ci.nodejs.org/computer/test-rackspace-win2012r2_vs2019-x64-3/
- https://ci.nodejs.org/computer/test-rackspace-win2012r2_vs2019-x64-5/

**Comment by richardlau** [comment]
I tried rebooting test-digitalocean-ubuntu1804_docker-x64-1 and test-digitalocean-ubuntu1804_docker-x64-2 but they don't appear to have come back online 😞.

**Comment by richardlau** [comment]
Restarted the agents on
- https://ci.nodejs.org/computer/test-softlayer-centos6-x64-1/
- https://ci.nodejs.org/computer/test-softlayer-centos6-x64-2/
- https://ci.nodejs.org/computer/test-digitalocean-debian8-x64-1/
- https://ci.nodejs.org/computer/test-rackspace-debian8-x64-1/
- https://ci.nodejs.org/computer/test-rackspace-debian8-x64-2/

**Comment by rvagg** [comment]
used "power cycle" in the DO dashboard on test-digitalocean-ubuntu1804_docker-x64-1 and test-digitalocean-ubuntu1804_docker-x64-2 and they are back up

Also did some quick maintenance on the Pi cluster and they are back online again and I've reenabled arm-fanned. I'm running some jobs through it now to fill up the workspaces I just emptied. there's 5 Pi1's marked offline and 4 Pi2's marked offline, nothing wrong with them (afaik) but I've done this to stop the workspaces from filling up unnecessarily. Especially with the Pi1's that are hardly ever run now. They're essentially "in reserve" for when other Pi's get knocked off. Pi 3's are all online because we have so much trouble keeping them up that I don't have much hope for the full suite staying afloat for long.
